packages/flutter/test/cupertino/sheet_test.dart

```diff
@@ -810,8 +810,7 @@ void main() {
     await tester.pumpAndSettle();
 
     final Finder clipRRectFinder = find.byType(ClipRRect);
-    final Rect clipRect = tester.getRect(clipRRectFinder);
-    expect(clipRect.center, equals(const Offset(400, 300)));
+    expect(clipRRectFinder, findsNothing);
```

Comment:
Why did this change exactly?

Ideally it would be nice if you could write a test that reproduces the error in https://github.com/flutter/flutter/issues/163954 and expect that it doesn't happen anymore.

Reply:
Since after popping this clipper doesn't exist anymore, that's why added this.
But yeah, it will make more sense to add another test which replicates error in #163954